examples/Introducing_CivisML_v2.ipynb

```diff
@@ -731,9 +738,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's plot the ROC curves and AUCs for each model. In the Civis Platform, these plots will automatically be built and displayed in the \"models\" tab. But for the sake of example, let's plot them here in the notebook.\n",
+    "Let's plot diagnostics for each of the models. In the Civis Platform, these plots will automatically be built and displayed in the \"Models\" tab. But for the sake of example, let's also explicitly plot ROC curves and AUCs in the notebook.\n",
     "\n",
-    "There are three classes (public, non-profit private, and for-profit private), so we'll have three curves per model. It looks like all of the models are doing well, with sparse logistic performing slightly worse."
+    "There are three classes (public, non-profit private, and for-profit private), so we'll have three curves per model. It looks like all of the models are doing well, with sparse logistic performing slightly worse than the other three."
```

Comment:
"than the other three" -> "than the other two"?

Reply:
Three-- GBT, RF, and stacking.